### PR TITLE
docs/week-8-announcement

### PR DESCRIPTION
**Pull request by ginge270**
# Description

This PR documents the Week 8 company announcement for Thoth Tech T1 2022

## Type of change

- [x] Documentation (update or new)

# How Has This Been Tested?

N/A

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Thanks for doing this Liz

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Week 8 Announcement
```
Comment (tancnle) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md]: **issue:** We will need to have a space between `#` and the header text, or else it won't render correctly.

```suggestion
# Week 8 Announcement
```

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Good Evening Thoth Tech
+
+It’s your friendly neighbourhood leadership team here.
+
+##RECAP:
```
Comment (tancnle) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md]:
```suggestion
## RECAP:
```

File: docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- Thank you to the 31 responses in the Onboarding feedback form. This will be invaluable information for next trimester. We are happy to report a 4/5 stars for on-boarding, hopefully next tri it will reach a 5/5 experience.
+- By now you should have all reviewed the all company meeting where we reviewed what we have achieved so far, what we still need to achieve and what new resources have become available since the beginning of trimester.
+
+##TO DO:
```
Comment (tancnle) [inline docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week8.md]:
```suggestion
## TO DO:
```

**Review by tancnle — COMMENTED** [review]
Thanks, @ginge270. I have left some suggestions for your consideration. Please let me know what you think 🙏🏼

**Comment by ghost** [comment]
Suggestions implemented. Ready to merge.

**Pull request merged**